### ISSUE TITLE
[Bug]: Full-text search does not perform reasonable range checking on the BM25 parameter.

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#36883]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:aa904be-dev
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):2.5.0rc96
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

valid range:
k:[0, inf)
b: [0,1]

when k,b's value was not in the right range, the index can still be created

index info:
[{'collection': 'full_text_search_collection_uo3jwXJW', 'field': 'emb', 'index_name': 'emb', 'index_param': {'index_type': 'HNSW', 'metric_type': 'L2', 'params': {'M': 16, 'efConstruction': 500}}}, {'collection': 'full_text_search_collection_uo3jwXJW', 'field': 'text_sparse_emb', 'index_name': 'text_sparse_emb', 'index_param': {'index_type': 'SPARSE_INVERTED_INDEX', 'metric_type': 'BM25', **'params': {'bm25_k1': -1, 'bm25_b': -1}**}}]

### Expected Behavior

creating index failed

**Comment by zhuwenxing** [comment]
/assign @zhengbuqian 
PTAL

**Comment by zhengbuqian** [comment]
/assign @zhuwenxing 
/unassign
#36907 has been merged, please verify

**Comment by zhuwenxing** [comment]
verified and fixed in `master-20750c0-20241017`
